domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/Claim.java

```diff
@@ -162,7 +162,10 @@ public Claim(
     }
 
     public LocalDate getServiceDate() {
```

Comment:
we might consider having an optional service date since it can be null

Reply:
Agree, that needs to be in separate PR. I changed that first but it was having bigger impact so left it for separate PR.

Reply:
👍